packages/odyssey-react-mui/src/labs/SideNav/SideNav.tsx

```diff
@@ -423,7 +433,7 @@ const SideNav = ({
   }, [isSideNavCollapsed, setSideNavCollapsed, onExpand, onCollapse]);
 
   const sideNavExpandKeyHandler = useCallback(
-    (event: KeyboardEvent<HTMLDivElement>) => {
+    (event: KeyboardEvent<HTMLButtonElement>) => {
```

Comment:
```useCallback<KeyboardEvent<HTMLButtonElement>>```

Reply:
I'm updating this but, for posterity, it needs to be `useCallback<KeyboardEventHandler<HTMLButtonElement>>`